app/src/shared/components/StorageNode/useStreamStorageNodeToggle.js

```diff
@@ -0,0 +1,71 @@
+import { useCallback, useState, useRef, useEffect } from 'react'
+import useIsMounted from '$shared/hooks/useIsMounted'
+import { post, del } from '$shared/utils/api'
+import routes from '$routes'
+
+const useStreamStorageNodeToggle = (streamId, address, _enabled, _changing) => {
```

Comment:
eslint should complain about `no-underscore-dangle`, no? Maybe `isEnabled` and `isChanging`